CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Updated `nghttp2` from `1.51.0` to `1.55.1`.
 - Converted SNP attestation UVM endorsements from integer to arbitrary string.
 - Updated Intel SGX PSW from 2.17 to 2.20 (#5616)
+- Added `/app/verifySnpAttestation` endpoint which verifies SNP Attestations. (#5653)
```

Comment:
That's in a test app, it's not the public API. This should instead document the introduction of ccf.SnpAttestation.verifySnpAttestation() as a public API for TypeScript apps.

Reply:
https://github.com/microsoft/CCF/pull/5653/commits/13bf121765a263ba2e1b4eede7eca5b3a8e4c4ac